bin/generatePhosimInput.py

```diff
@@ -0,0 +1,129 @@
+    """
+    sql_string = 'SELECT * FROM Summary WHERE ObsHistID in ('
+    sql_string += ', '.join(map(str, obsHistIDList))
+    sql_string += ')'
+    return sql_string
+
+
+def generateSinglePointing(obs_metaData, availableConns, sntable,
+                           fname,
+                           sn_sed_file_dir):
+    """
+    obs_metaData : instance of `lsst.sims.utils.ObservationMetaData`
+        observation metadata corresponding to an OpSim pointing
+    availableConns : available connections to fatboy
+    sntable : Table for SN on the fatboy database
+    fname : output file for phoSim instance Catalog
+    sn_sed_file_dir : directory to which the SN seds corresponding to this
+        phoSim metadat
+    """
+    tstart = time.time()
+    obs_metaData.boundLength = 0.3
+    print(obs_metaData.summary)
+    obsHistID = obs_metaData._OpsimMetaData['obsHistID']
+
+    # all but first two are default values of optional parameters
+    # Kept in script to emphasize inputs
+    tSky = TwinklesSky(obs_metadata=obs_metaData,
+                       availableConnections=availableConns,
+                       brightestStar_gmag_inCat=11.0,
+                       brightestGal_gmag_inCat=11.0,
+                       sntable=sntable,
+                       sn_sedfile_prefix=os.path.join(sn_sed_file_dir, 'specFile_'))
+    # fname = phoSimInputFileName(obsHistID)
+    # if not os.path.exists(os.path.dirname(fname)):
+    #    os.makedirs(os.path.dirname(fname))
+    if not os.path.exists(sn_sed_file_dir):
+        os.makedirs(sn_sed_file_dir)
+    tSky.writePhoSimCatalog(fname)
+    availConns = tSky.availableConnections
+    tend = time.time()
+    print (obsHistID, tend - tstart)
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(description='Write phoSim Instance Catalogs'
+                                     'and SN spectra to disk '
+                                     'example : '
+                                     'python generatePhosimInput.py 230\ '
+                                     '--OpSimDBDir ~/data/LSST/OpSimData/\ '
+                                     '--seddir "./"\ '
+                                     '--outfile phosim_instance_catalog_220.txt')
+    parser.add_argument('--opsimDB',
+                        type=str,
+                        help='OpSim database sqlite filename',
+                        default='minion_1016_sqlite.db')
+    parser.add_argument('visit',
+                        type=int,
+                        help='Visit number (obsHistID)')
+    parser.add_argument('--OpSimDBDir',
+                        help='absolute path to dir with the opsimBD',
+                        type=str,
+                        default='./')
+    parser.add_argument('--outfile', type=str, default=None,
+                        help='output filename for instance catalog')
+    parser.add_argument('--seddir',
+                        type=str,
+                        default=None,
```

Comment:
I think this line would be better as
```
                       default='.',
```
so that line 120 below works in the default case. (I should have noted this change and the above in my example code when I made my last comment on the command line interface.)